src/csharp/Grpc.IntegrationTesting/InteropClient.cs

```diff
@@ -577,7 +580,16 @@ public static void RunJwtTokenCreds(TestService.TestServiceClient client)
             Console.WriteLine("Passed!");
         }
 
-        public static void RunUnimplementedMethod(UnimplementedService.UnimplementedServiceClient client)
+        public static void RunUnimplementedService(UnimplementedService.UnimplementedServiceClient client)
+        {
+            Console.WriteLine("running unimplemented_service");
+            var e = Assert.Throws<RpcException>(() => client.UnimplementedCall(new Empty()));
+
+            Assert.AreEqual(StatusCode.Unimplemented, e.Status.StatusCode);
+            Console.WriteLine("Passed!");
+        }
+
+        public static void RunUnimplementedMethod(TestService.TestServiceClient client)
         {
             Console.WriteLine("running unimplemented_method");
             var e = Assert.Throws<RpcException>(() => client.UnimplementedCall(new Empty()));
```

Comment:
btw it looks like theres a slight mismatch between the interop descriptions and the service/method name for unimplemented_method, with it being client.UnimplementedMethod in the description, but I'm not sure if this was working before?